pantheon/src/main/java/tech/pegasys/pantheon/cli/PantheonCommand.java

```diff
@@ -658,6 +678,19 @@ public void run() {
     }
   }
 
+  private void checkFastSyncOptions() throws ParameterException {
+    if (fastSyncMaxWaitTime < 0) {
+      throw new ParameterException(
+          commandLine, "--fast-sync-max-wait-time must be a positive or zero integer");
+    }
+    checkOptionDependencies(
```

Comment:
This check option dependencies should be moved to be with the others rather than in its own method.

Reply:
Ok. I was just thinking that the other method will become bigger and bigger.

Reply:
It will.  When we do something different we should be consistent and do it differently for all, not just new arriving code. But fixing the size problem is out of scope for this PR.